week-4/A-objects-intro/README.md

```diff
@@ -1,8 +1,23 @@
-### Goals
-
-- introduce objects as a concept
-- when can we use objects (to store what type of information)
-- the very specific object syntax (the commas after each property, the colon between property name and value, the curly brackets)
-- good practices for naming giving names to properties (not having spaces, what to do when there is a space)
-- the language to use when talking about an objects: object literal, properties, property names, property values
-- very simple exercises to get them used to declaring object literals
+### Objects
+
+When you think about real world objects, they have a bunch of properties that describe how its unique. Your laptop, for example, has a brand (Lenovo/Apple etc.), a screen size (13/15 inch), RAM (8/16GB) etc.
+
+As arrays in JavaScript are useful for describing a collection of things, objects are very useful for grouping together a bunch of characteristics to describe a specific entity.
+
+How would we describe the above laptop as a JavaScript object?
+
+```js
+var laptop = {
+    brand: "Lenovo",
+    screenSize: 13,
+    isTouchscreen: true
+};
+```
+
+What's happening above is that we're **declaring a variable called laptop** and assigning to it...an **object literal**! 
+
+An object literal is composed of a set of curly bracket `{...}`, and in between a set of properties that describe that object: `brand`, `screenSize`, `isTouchScreen`.

```

Comment:
I've avoided describing syntax in the other lessons. I think the best description of the syntax is an example of the code itself.

Reply:
It's a short lesson without the description, but I think that's totally fine! Students seem to adverse to reading long descriptions.

Reply:
You're right. I thought that this being such a simple lesson, just an intro on objects, I could at least focus on the syntax and the language to use.

However, there's no real need to write that down.